src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -249,7 +689,28 @@ IMF_STD_ATTRIBUTE_DEF (timeCode, TimeCode, TimeCode)
 
 
 //
-// wrapmodes -- determines how texture map images are extrapolated.
+// timecodeRate -- the timecode rate of the image sequence to which
+// the image belongs, specified in timecodes per second
+//
+
+IMF_STD_ATTRIBUTE_DEF (timecodeRate, TimecodeRate, int)
```

Comment:
is timecodeRate associated with the framesPersecond (playback rate) or the rate (capture rate)? I think whichever it is, timecodeRate should be in the file near by, and there should be a note to clarify that association. Either way, those values are rational numbers expressing for example 1/24. Does the timecodeRate tell us that the timecode for example, increments by 10, for each image? Where is the timecode itself encoded? I assume this timecode does not correlate to a SMPTE timecode? Or is the implicit 24, 30, 29.95 of the SMPTE time code? In that case shouldn't the timecodeRate itself be a rational to express eg. 3000/1001?

Reply:
I'm almost certain that timecodeRate has to do with the timecode that is recorded at scene capture; that is is 'number of timecode values assigned per second'. It's always bugged me that we say 'drop-frame timecode' rather than 'sometimes-skipping timecode' but I do see how hard it is to name the behavior. I will ask Lars about this.

Reply:
And SMPTE timecode have got an additional bit of goofiness which is punctuation is the only way to tell the difference between a drop frame and non-drop frame code. Can 1950's TV engineering please take a well deserved bow and reconvene in the lounge for some whiskey so we can get it on with other things 😬